pepper-apis/studybuilder-cli/studies/singular/patches/singular-race-question-prompt-fix.conf

```diff
@@ -0,0 +1,3 @@
+{
+  "prompt": "What is $ddp.checkAnswer(\"PATIENT_SURVEY_WHO_ENROLLING_HIDDEN\",\"CHILD\", \"your child's\", \"your\") race?*"
```

Comment:
@MocanaAtBroad 
Please apply this template change to i18n/en.conf too.
If not whenever we run UpdateTemplatesInPlace change will be lost.

Reply:
9d605b24f142b73ccba51322ac9ee81ae9b1ce45